proposals/unified-site.md

```diff
@@ -0,0 +1,163 @@
+itself. We need a place where we talk with a unified voice about how it's
+different, what its benefits are, why you would and wouldn't use it, how it
+compares to Web2, what good metaphors are in the current world for where we
+envision things going, and how the tech works together to address common
+scenarios in a unique way.
+
+Onboarding is an art, and creating a golden path through our offerings that
+shows various pieces of tech working together will require all of us pulling on
+the same oar, not continuing to maintain our silos.
+
+To continue to sink development time into any one particular website - such as,
+say, ipfs.io - is to create further inconsistency between experiences among our
+sites, and takes valuable development time away from dealing with our key
+problems: that we have too much to maintain and things are going stale, and we
+have no place to paint the whole picture and make the value clear. Symptoms of
+these core issues cropped up again and again in discovery.
+
+To address an elephant in the room: we don't have to stamp "PL" onto a unified
+site. It can be at "w3dt.io," for example, and we can maintain an "About" page
+that talks about the team at PL that is tasked with the mission of bringing this
+tech to the world, as well as community and partner contributions and
+strenuously highlight that the tech consists of a collection of open-source
+projects. However, if the time has come to acknowledge these as PL projects, I
+think unifying the docs with protocol.ai would be fine, too.
+
+#### Assumptions &amp; hypotheses
+
+- There are too many repos
+- Sites are in a various state of maintenance
+- We are not clearly stating how tech works together
+- There is no easy path to upgrade all of our websites at once
+- There is very little "lift" from one technology being popular translating to another technology gaining in adoption
+- There is no place where we are onboarding people onto our stack as a whole or building a general W3 mental model
+- We are sinking time into maintaining siloed projects and there is an opportunity cost inherent there
+
+#### User workflow example
+
+They would load this website. At first I presume we would have a simple project list, sort of like https://www.cncf.io/sandbox-projects/
+
+In the long term we should build up scenario-based onboarding that is more
+geared towards building the mental model of how the stack solves problems, such
+as what you see at https://www.digitalocean.com/business/
+
+#### Impact
+
+🔥🔥🔥
+
+How can we even talk about the "web3 dev stack product-market fit" without a
+website that is dedicated to it, or without even regarding it as a stack in the
+first place?
+
+#### Leverage
+
+🎯🎯🎯
+
+The amount of distraction that would be avoided alone by trying to do this
+across multiple websites, projects, and repos justifies the project, but if we
+actually wnat to start convincing people that there is such a thing as a "web3
```

Comment:
I don't think convincing people there is a web3 dev stack is an overt goal - our goal is to help people use the protocols we work on and use them together effectively. I don't see the evidence in this proposal that decreases distraction across multiple websites (this would be additive, adding a new site that actually increases distraction)